src/main/java/org/example/service/CustomerInputService.java

```diff
@@ -61,11 +66,11 @@ public String getDateOfBirth(){
             } else{
                 System.out.println("invalid input year");
             }
-        }catch (NumberFormatException e){
-            System.out.println(e);
+        }catch (Exception e){
```

Comment:
this is wrong to have generic exception,
initially you should catch actual exception then in the last generic if needed